crates/ruff_linter/resources/test/fixtures/flake8_pyi/PYI030.py

```diff
@@ -36,3 +36,54 @@ def func2() -> Literal[1] | Literal[2]:  # Error
 
 # Should emit for union in generic parent type.
 field11: dict[Literal[1] | Literal[2], str]  # Error
+
+# Should emit for unions with more than two cases
+field12: Literal[1] | Literal[2] | Literal[3]  # Error
+field13: Literal[1] | Literal[2] | Literal[3] | Literal[4]  # Error
+
+# Should emit for unions with more than two cases, even if not directly adjacent
+field14: Literal[1] | Literal[2] | str | Literal[3]  # Error
+
+# Should emit for unions with mixed literal internal types
+field15: Literal[1] | Literal["foo"] | Literal[True]  # Error
+
+# Shouldn't emit for duplicate field types with same value; covered by Y016
+field16: Literal[1] | Literal[1]  # OK
```

Comment:
The only current outstanding issue is that the test files for `PYI030` suggest that the rule should not emit on this line here, but at the moment it still does on the `main` branch.

What do we want to do here?

Reply:
👍 no problem